### PR TITLE
finish supervisor move trainees out of course

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

File: app/views/rails_admin/main/show_course.html.erb [hunk 1]
```diff
@@ -36,7 +36,7 @@
         <dd class="well">
           <%= @course.description %>
         </dd>
```
Comment (trantube94) [inline app/views/rails_admin/main/show_course.html.erb]: add space

**Comment by vietanhbkaaa** [comment]
OK

File: app/views/rails_admin/main/show_course.html.erb [hunk 2]
```diff
@@ -48,11 +48,41 @@
...
+              </td>
+              <td class="last links">
+                <ul class="inline list-inline">
+                  <li>
+                    <%= link_to "", rails_admin.show_path(User, member.id),
+                      class: "icon-info-sign",
+                      title: t("admin.actions.show_member.title") %>
+                  </li>
+                  <li>
+                    <%= link_to "", rails_admin.delete_path(UserCourse, member.user_courses.find_by(course: @course)),
```
Comment (vietanhbkaaa) [inline app/views/rails_admin/main/show_course.html.erb]: nghi vấn ?

**Comment by trantube94** [comment]
OK

**Pull request merged**